packages/manager/apps/pci-kubernetes/src/routes.tsx

```diff
@@ -125,13 +125,13 @@ export default [
               {
                 path: 'add-oidc-provider',
                 ...lazyRouteConfig(() =>
-                  import('@/pages/oidc-provider/AddOIDCProvider.page'),
+                  import('@/pages/oidc-provider/AddOrUpdateOIDCProvider.page'),
                 ),
               },
               {
                 path: 'update-oidc-provider',
                 ...lazyRouteConfig(() =>
-                  import('@/pages/oidc-provider/UpdateOIDCProvider.page'),
+                  import('@/pages/oidc-provider/AddOrUpdateOIDCProvider.page'),
```

Comment:
**nitpick:** Each page/route must be a folder according to the defined folder structure.Can you please correct it?
```suggestion
                  import('@/pages/oidc-provider/update/AddOrUpdateOIDCProvider.page'),
```